tests/test_TFNetworkRecLayer.py

```diff
@@ -4161,6 +4161,118 @@ def test_trafo_search_lm():
       assert all(out_seqs[i, :input_seq_lens[i]] == input_seqs[i, :input_seq_lens[i]])
 
 
+def test_offset_no_rec_positional_encoding():
```

Comment:
The function should be called `test_ PositionalEncodingLayer_offset_no_rec` or so.

Reply:
Done.